sdk/modelsrepository/azure-iot-modelsrepository/src/samples/README.md

```diff
@@ -0,0 +1,198 @@
+# IoT Models Repository Samples
+
+The Azure IoT Models Repository enables builders to manage and share digital twin models for global consumption. The models are [JSON-LD][json_ld_reference] documents defined using the Digital Twins Definition Language ([DTDL][dtdlv2_reference]).
+
+For more info about the Azure IoT Models Repository checkout the [docs][modelsrepository_msdocs].
+
+## Introduction
+
+You can explore the models repository APIs with the client library using the samples project.
+
+The samples project demonstrates the following:
+
+- Instantiate the client
+- Get models and their dependencies from either a remote endpoint or local repository.
+
+## Initializing the models repository client
+
+```java
+// When no URI is provided for instantiation, the Azure IoT Models Repository global endpoint
+// https://devicemodels.azure.com/ is used and the model dependency resolution
+// configuration is set to TryFromExpanded.
+ModelsRepositoryClientBuilder clientBuilder = new ModelsRepositoryClientBuilder();
+ModelsRepositoryAsyncClient asyncClient = clientBuilder.buildAsyncClient();
+ModelsRepositoryClient syncClient = clientBuilder.buildClient();
+
+System.out.println("Initialized the async client pointing to the global endpoint" + asyncClient.getRepositoryUri().toString());
+System.out.println("Initialized the sync client pointing to the global endpoint" + syncClient.getRepositoryUri().toString());
+```
+
+```java
+// The client will also work with a local file-system URI. This example shows initialization
+// with a local URI and disabling model dependency resolution.
+clientBuilder
+    .repositoryEndpoint(LOCAL_DIRECTORY_URI)
+    .modelDependencyResolution(ModelDependencyResolution.DISABLED);
+asyncClient = clientBuilder.buildAsyncClient();
+syncClient = clientBuilder.buildClient();
+
+System.out.println("Initialized the async client pointing to the local file-system: " + asyncClient.getRepositoryUri().toString());
+System.out.println("Initialized the sync client pointing to the local file-system: " + syncClient.getRepositoryUri().toString());
+```
+
+## sync vs async clients
```

Comment:
We aren't referencing reserved keywords, so why not capitalize the S in sync?

Reply:
good point, changing.